src/CommandLine/Endpoint.cs

```diff
@@ -36,5 +36,29 @@ public static async Task Create(ManagementClient client, CommandArgument name, C
                 Console.WriteLine("Subscription already exists, skipping creation");
             }
         }
+
+        public static async Task Subscribe(ManagementClient client, CommandArgument name, CommandOption topicName, CommandOption subscriptionName, CommandArgument eventType, CommandOption ruleName)
+        {
+            try
+            {
+                await Rule.Create(client, name, topicName, subscriptionName, eventType, ruleName);
+            }
+            catch (MessagingEntityAlreadyExistsException)
+            {
+                Console.WriteLine($"Rule already exists, skipping creation. Verify SQL filter matches '[NServiceBus.EnclosedMessageTypes] LIKE '%{eventType.Value}%'.");
+            }
+        }
+
+        public static async Task Unsubscribe(ManagementClient client, CommandArgument name, CommandOption topicName, CommandOption subscriptionName, CommandArgument eventType, CommandOption ruleName)
+        {
+            try
+            {
+                await Rule.Delete(client, name, topicName, subscriptionName, eventType, ruleName);
+            }
+            catch (MessagingEntityAlreadyExistsException)
```

Comment:
Is this the right exception to to be catching here? I think it needs to be `MessagingEntityNotFoundException`, but it would be worth verifying (add a test?)

Reply:
I think you're right.

Reply:
Fixed